tests/acceptance/pageObjects/phoenixPage.js

```diff
@@ -231,6 +247,14 @@ module.exports = {
     acceptSharesInNotifications: {
       selector: '//div[@id="oc-notification"]//h5[contains(text(),\'%s\')]/../div/button/span[.="Accept"]',
       locateStrategy: 'xpath'
+    },
+    userProfile: {
+      selector: '//div/button[@id="_userMenuButton"]',
```

Comment:
```suggestion
      selector: '._userMenuButton',
      locateStrategy: 'css selector'
```

Reply:
Or, even better, check if there's any other selectors that can be used (that doesnot start with `_` (underscore))